### ISSUE TITLE
Mock Signup Page

### ISSUE THREAD

**Issue by CraigBryan** [CraigBryan/meeto-mock#2]

**Comment by CraigBryan** [comment]
In reference to Dan's commit for the signup form. He was asking about how to do the availability as a popup or standard time picker or something else.

Is it using a standard time picker now? Is the availability suppose to be just time? 

I think a nice solution would be a nice, big, central dialog-style popup. Have it gray out the background, and have clicking an X in the corner or anywhere outside of the pop-up close the dialog. If we find/write a nice bit of javascript that does this, we can use it consistently for more complex choices the user has to make. Does that make sense?

**Comment by charle692** [comment]
The availability as I made it was a custom dialog which allowed you to set your availability for the weekend and the weekdays separately. I'm not sure if we would like to make that per day instead?

**Comment by DanielMurdoch** [comment]
The bootstrap modal is exactly what you described Craig. We can use that,
and put some JavaScript time picker inside of it.

**Comment by CraigBryan** [comment]
Great. Works for me.

**Comment by charle692** [comment]
Ok are you guys opposing to using the pencil glyphicon instead of "Edit"? Also are you guys ok with me changing the buttons to the default color. I find that it's hard to tell if something is a button with the btn-default.

**Comment by DanielMurdoch** [comment]
Edit I don't care. Buttons I really hate how brash the primary colour is :/
I definitely prefer the default but if both of you don't like it then shoot,

**Comment by charle692** [comment]
ok no that's fine we can keep it the same. Will make sure to change all of my buttons to the default so that were consistent.

**Comment by CraigBryan** [comment]
I agree that the primary is too bright. I'm using the pencil glyphicon I believe.... Unless you are using another one. Look at profile.html

**Comment by charle692** [comment]
yeah that's a different one but either one is fine. The pencil icon is just a pencil with no square.

**Comment by charle692** [comment]
ok well right now were using btn-primary for all of the modals should we change that?

**Comment by DanielMurdoch** [comment]
Think we have too? They're modals vs like actual page buttons, so I don't
know if they HAVE to be different, will he care? If so then sure!

**Comment by charle692** [comment]
You mean don't have to right? And not sure I guess we could just ask him tmrw and see what he thinks. I think the demo is more for feedback than anything.

**Comment by DanielMurdoch** [comment]
Okay that's cool, either way i don't really have an opinion!

**Comment by CraigBryan** [comment]
I changed my glyphicon to pencil. Consistency is GOD!

**Comment by CraigBryan** [comment]
FYI, I was curious, and the demo is worth 15% of the project mark.

**Comment by charle692** [comment]
What!?

**Comment by CraigBryan** [comment]
"implementation and demo", "15%", "March 30th"
Its in the project phase 4 pdf at the top.

**Comment by charle692** [comment]
:no_good:

**Comment by DanielMurdoch** [comment]
Yeah i already knew that aha, and 15% of the project mark, not the final mark, so this demo is only worth 3.75% final mark. Yeah. The report and in class thing are worth a significant amount more combined aha, hell each lab was worth almost the same amount as this (3%).

**Comment by CraigBryan** [comment]
Yeah, I'm not terribly concerned about it. I was just curious because Ryan seemed to think it wasn't worth anything.

**Comment by CraigBryan** [comment]
At the top of the signup page there some whitespace, but none below. It just looks a little unsymmetrical.

**Comment by charle692** [comment]
Yeah I'm going through it right now and fixing stuff like that.

**Comment by DanielMurdoch** [comment]
Yeah that is some random bullshit heading whitespace that bootstrap automatically puts in. I manually fixed it on some other pages, but not that one i guess.

**Comment by CraigBryan** [comment]
Cool. I'm doing a bit of work on the intro. I'll put some actual words in.

**Comment by DanielMurdoch** [comment]
Sweet thanks!
